.github/workflows/meds.yml

```diff
@@ -0,0 +1,26 @@
+name: MEDs docker
+
+# Controls when the action will run. Triggers the workflow on release 
+# event
+
+on:
+  push:
+    branches: [meds\dev]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+
+  webhook:
+    name: Send event
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set platform repository mame
```

Comment:
You can simplify the environment variable declaration like here https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#about-environment-variables

Reply:
Fixed